lib/livebook/hubs.ex

```diff
@@ -0,0 +1,11 @@
+defprotocol Livebook.Hubs do
```

Comment:
If the wording is "a protocol for loading hubs from somewhere", we can use a more specific name for the protocol, like `Livebook.HubProvider` or similar.

Reply:
And perhaps it's worth moving it to the `Hubs.` namespace as well.

Reply:
I like this one